.github/ISSUE_TEMPLATE/SUPPORT_QUESTION.md

```diff
@@ -1,13 +1,12 @@
 ---
 name: 💬 Support Question
 about: If you have a question, please check out our Slack or StackOverflow!
-
 ---
 
-<!------------^ Click "Preview" for a nicer view! -->
+<!-- Click "Preview" for a nicer view! -->
 
-Apache Cordova uses GitHub Issues as a feature request and bug tracker _only_.
-For usage and support questions, please check out the resources below. Thanks!
+Apache Cordova uses GitHub Issues _only_ for feature requests and tracking issues.
```

Comment:
Everything in GitHub issues are issues. Maybe:
```suggestion
Apache Cordova uses GitHub Issues _only_ for feature requests and tracking bugs.
```

Reply:
I feel it's better to use a more general term here.